network/p2p/libp2pUtils.go

```diff
@@ -163,6 +163,10 @@ func generateProtocolID(rootBlockID string) protocol.ID {
 	return protocol.ID(FlowLibP2PProtocolIDPrefix + rootBlockID)
 }
 
+func generateID(rootBlockID string) protocol.ID {
```

Comment:
```suggestion
func generatePingID(rootBlockID string) protocol.ID {
```
Is this only for ping messages?

Reply:
yeah  only for Ping message. I believe my earlier name change got lost somehow. I have added it back.